src/main/java/seedu/foodrem/logic/commands/tagcommands/UntagCommand.java

```diff
@@ -0,0 +1,101 @@
+package seedu.foodrem.logic.commands.tagcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.foodrem.logic.parser.CliSyntax.PREFIX_ID;
+import static seedu.foodrem.logic.parser.CliSyntax.PREFIX_TAG_NAME;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.foodrem.commons.core.index.Index;
+import seedu.foodrem.logic.commands.Command;
+import seedu.foodrem.logic.commands.CommandResult;
+import seedu.foodrem.logic.commands.exceptions.CommandException;
+import seedu.foodrem.model.Model;
+import seedu.foodrem.model.item.Item;
+import seedu.foodrem.model.tag.Tag;
+
+/**
+ * Untags an item with a Tag.
+ */
+public class UntagCommand extends Command {
+
+    public static final String COMMAND_WORD = "untag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Untags the item identified by the index number used in the displayed item list with a valid Tag.\n"
+            + "Parameters: " + PREFIX_TAG_NAME + "TAG_NAME " + PREFIX_ID + " INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_TAG_NAME + "Condiments " + PREFIX_ID + "1";
+
+    public static final String MESSAGE_SUCCESS = "Item untagged successfully";
+
+    public static final String ITEM_NOT_TAGGED = "This item is not tagged with this tag";
+
+    public static final String MESSAGE_TAG_DOES_NOT_EXIST = "This tag does not exist";
+
+    public static final String MESSAGE_ITEM_INDEX_DOES_NOT_EXIST = "The item index does not exist";
+
+    private final Index index;
+    private final Tag tag;
+
+    /**
+     * Creates a TagCommand to tag the specified {@code Item} with a specified {@code Tag}
+     */
+    public UntagCommand(String tagName, Index index) {
+        requireNonNull(tagName);
+        requireNonNull(index);
+        this.index = index;
+        this.tag = new Tag(tagName);
+    }
+
+    /**
+     * Creates and returns a {@code Item} with the tagSet of {@code itemToUntag}
+     * edited
+     */
+    private static Item createUntaggedItem(Item itemToUntag, Tag tag) {
+        assert itemToUntag != null;
+
+        itemToUntag.removeTag(tag);
```

Comment:
Question regarding the implementation: I see that removeTag removes the existence of that tag from FoodRem "Removes {@code key} from this {@code AddressBook}". Will calling this cause bugs where all references to that tag will be removed?

Reply:
This method call to removeTag refers to the method implemented in the Item class (not the FoodRem class). But yes, I think it's better to rename the method to removeItemTag instead, for clarity.